src/scss/_common.scss

```diff
@@ -2,11 +2,19 @@
 * General Uppy styles that apply to everything inside the .Uppy container
 */
 
-.Uppy {
+.uppy {
```

Comment:
why this caps change?

Reply:
now i read further down, I guess this sorta makes sense for `uppy-PluginName` style classnames. if `uppy` is a namespace it might make sense to use `uppy-Root` for the classname for the root element imho (like in BEM you don't have bare namespace classes) but that's a nitpick

Reply:
Yeah, namespace. I was looking here, among other places, https://github.com/codemirror/CodeMirror/blob/master/lib/codemirror.css. Thinking if we need “root” or something like that, and also `c-` or `u-` prefixes for things like `UppyIcon`.

Reply:
Also initially we were aiming to follow this: https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md